infrastructure/resource_group_level/modules/portal-api.bicep

```diff
@@ -26,14 +27,19 @@ param additionalSecureSettings object = {
   DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
 }
 
+// These values are inlined in the CLI scripts
+param connectionStrings object = {
+  COMPANIES_HOUSE_API_URL: 'test-value' // from env
+}
+
 // These values are taken from GitHub secrets injected in the GHA Action
 @secure()
 param secureConnectionStrings object = {
   // NOTE that CORS_ORIGIN is not present in the variables exported from dev or staging
   CORS_ORIGIN: 'test-value'
   AZURE_PORTAL_EXPORT_FOLDER: 'test-value'
   AZURE_PORTAL_FILESHARE_NAME: 'test-value'
-  // TODO:FN-737 *_WORKFLOW_* variables are not needed and can be removed.
+  // TODO:FN-749 Confirm *_WORKFLOW_* variables are not needed and can be removed.
```

Comment:
they are definitely not used :) 
`Workflow` was an old system that we were supposed to integrate with. this was replaced with TFM

Reply:
Good stuff - I've updated FN-749 with this comment and will strip them in that so we're consistent with the changes on `main`.